warehouse/integrations/manager/manager.go

```diff
@@ -26,6 +26,9 @@ import (
 	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
 )
 
+// Creating an alias since "model.TableSchema" is defined in an internal module
+type ModelTableSchema = model.TableSchema
```

Comment:
What is the alternative ? I can't import `model.TableSchema` in the snowpipestreaming test since it is in an internal module.

Reply:
We can use whutils.ModelTableSchema